css/style.css

```diff
@@ -138,27 +131,82 @@ nav {
   float: right;
 }
 
-.content {
+section {
+  background: #f0f0f0;
+}
+
+.icons {
+  width: 33%;
+  border: 1px solid #f0f0f0;
+  height: 287px;
+  background-color:blue;
+  float: left;
+}
+
+.grid {
+  padding-top: 80px;
+  padding-bottom: 80px;
   background: #f0f0f0;
-  height: 575px;
 
 }
 
+.grid p {
+    text-transform: uppercase;
+    font-size: 15px;
+    padding-bottom: 10px;
+    letter-spacing: .3em;
+}
+
+.grid h1 {
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 35px;
+  margin-bottom: 20px;
+  letter-spacing: .4em;
+}
+
+.icons p {
+  color: rgba(255, 255, 255, 0.7);
+  text-shadow: 0 1px 1px rgba(0, 0, 0, 0.5);
+  font-weight: bold;
+  text-align: center;
+  padding-top: 30px;
+}
+
+img.main-icons{
+  display: block;
+  margin: 0px auto;
+  padding-top: 60px;
+  opacity: .25;
+}
+
 footer {
-  background: #3FBFBF;
-  height: 65px;
-  color: white;
+  background: black;
+  height: 100px;
   font-size: 1em;
 }
 
-li {
+footer ul {
   float: right;
-  padding-top: 25px;
-  padding-left: 125px;
 }
 
-.logo-small {
-  padding-top: 18px;
-  float: left;
+footer li {
+  display: inline-block;
+  padding: 45px 30px;
+}
+
+@media (max-width: 667px) {
+
+.icons {
+  width: 100%;
+  float: none;
+}
```

Comment:
nice! look at how easy it was to make this site mobile. most of your layout just takes care of itself -- and it was very very few lines of code to make it all shrink and look good. this is the yield of good foresight when making your desktop layout